source/core/output_buffers/ut_output_buffer_tmp.sql

```diff
@@ -25,9 +30,18 @@ create table ut_output_buffer_tmp$(
   constraint ut_output_buffer_tmp_pk primary key(output_id, message_id),
   constraint ut_output_buffer_tmp_ck check(is_finished = 0 and text is not null or is_finished = 1 and text is null),
   constraint ut_output_buffer_fk1 foreign key (output_id) references ut_output_buffer_info_tmp$(output_id)
-) organization index overflow nologging initrans 100
-  lob(text) store as securefile ut_output_text(retention none)
-;
+) organization index overflow nologging initrans 100 ';
+  begin
+    execute immediate
+      v_table_sql || 'lob(text) store as securefile ut_output_text(retention none)';
+  exception
+    when e_non_assm then
+      execute immediate
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a WHEN OTHERS exception handler to ensure that all exceptions are trapped. ([plsql:ExecuteImmediateTrapExceptionsCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AExecuteImmediateTrapExceptionsCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=816&issues=AWhCR0RdSi5GWQS9iTtf&open=AWhCR0RdSi5GWQS9iTtf)

[ISSUE_KEY]: AWhCR0RdSi5GWQS9iTtf